src/lfortran/semantics/comptime_eval.h

```diff
@@ -840,6 +837,9 @@ TRIG2(sqrt, dsqrt)
             Str s;
             s.from_str_view(svalue);
             char *str_val = s.c_str(al);
+            // TODO: Should be 0 for char(0) but we store it as 1
+            ASR::ttype_t* str_type = LFortran::ASRUtils::TYPE(ASR::make_Character_t(al,
+                loc, 1, 1, nullptr, nullptr, 0));
```

Comment:
```suggestion
            ASR::ttype_t* str_type = LFortran::ASRUtils::TYPE(ASR::make_Character_t(al,
                loc, 1, strlen(str_val), nullptr, nullptr, 0));
```

Shouldn't we keep the length because we already know?

Reply:
`char` always returns a character of length 1. See, https://gcc.gnu.org/onlinedocs/gfortran/CHAR.html